### ISSUE TITLE
Conan Error Installing - Cannot load recipe

### ISSUE THREAD

**Issue by mraburn** [ripple/validator-keys-tool#52]
Seems the recipe is calling xrpl/2.1.0 does not exist.

`Recipe 'xrpl/2.1.0' seems broken.
It is possible that this recipe is not Conan 2.0 ready
If the recipe comes from ConanCenter, report it at https://github.com/conan-io/conan-center-index/issues
If it is your recipe, check if it is updated to 2.0`

**Comment by ximinez** [comment]
That is correct. You can add the Ripple remote to download it: https://github.com/ripple/validator-keys-tool/blob/master/README.md#build

**Comment by derekneves** [comment]
Getting the same using conan 2.8.0
`root@149e2ee21a23:~# git clone https://github.com/ripple/validator-keys-tool.git
Cloning into 'validator-keys-tool'...
remote: Enumerating objects: 255, done.
remote: Counting objects: 100% (126/126), done.
remote: Compressing objects: 100% (57/57), done.
remote: Total 255 (delta 87), reused 83 (delta 67), pack-reused 129 (from 1)
Receiving objects: 100% (255/255), 103.81 KiB | 165.00 KiB/s, done.
Resolving deltas: 100% (140/140), done.
root@149e2ee21a23:~# cd validator-keys-tool/
root@149e2ee21a23:~/validator-keys-tool# mkdir .build
root@149e2ee21a23:~/validator-keys-tool# cd .build
root@149e2ee21a23:~/validator-keys-tool/.build# conan install .. --output-folder . --build missing

======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=11
os=Linux

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=11
os=Linux


======== Computing dependency graph ========
xrpl/2.1.0: Not found in local cache, looking in remotes...
xrpl/2.1.0: Checking remote: conancenter
xrpl/2.1.0: Checking remote: ripple
xrpl/2.1.0: Downloaded recipe revision 0
Graph root
    conanfile.txt: /root/validator-keys-tool/conanfile.txt
*********************************************************
Recipe 'xrpl/2.1.0' seems broken.
It is possible that this recipe is not Conan 2.0 ready
If the recipe comes from ConanCenter, report it at https://github.com/conan-io/conan-center-index/issues
If it is your recipe, check if it is updated to 2.0
*********************************************************

ERROR: Package 'xrpl/2.1.0' not resolved: xrpl/2.1.0: Cannot load recipe.
Error loading conanfile at '/root/.conan2/p/xrpl6830fce0a89a0/e/conanfile.py': Error while initializing options. The usage of package names `cassandra-cpp-driver:shared` in options is deprecated, use a pattern like `cassandra-cpp-driver/*:shared` instead.
root@149e2ee21a23:~/validator-keys-tool/.build# conan --version
Conan version 2.8.0
root@149e2ee21a23:~/validator-keys-tool/.build# `
